translations/ru.json

```
@@ -0,0 +1,27 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "Устройство уже настроено"
+    },
+    "error": {
+      "cannot_connect": "Не возможно подключиться к серверу Photoprism.",
+      "invalid_auth": "Неверная аутентификация",
+      "unknown": "Неизвестная ошибка",
+      "already_configured": "Сервис уже настроен"
+    },
+    "step": {
+      "user": {
+        "title": "Настройка интеграции Photoprism",
+        "data": {
+          "name": "Имя",
+          "url": "URL",
+          "username": "Имя пользователя",
+          "password": "Пароль",
+          "port": "Порт",
+          "verify_ssl": "Проверить SSL"
```

```suggestion
          "verify_ssl": "Проверять SSL"
```
Мне кажется логичнее, это не однократная проверка